netbeans/CheckerFrameworkPlugin/src/org/checkerframework/netbeans/CheckerFrameworkPanel.java

```diff
@@ -0,0 +1,135 @@
+package org.checkerframework.netbeans;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import javax.swing.BoxLayout;
+import javax.swing.JCheckBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import org.netbeans.api.project.ProjectManager;
+import org.netbeans.spi.project.support.ant.EditableProperties;
+import org.openide.filesystems.FileObject;
+import org.openide.filesystems.FileUtil;
+import org.openide.modules.InstalledFileLocator;
+import org.openide.util.MutexException;
+
+public class CheckerFrameworkPanel extends JPanel {
+
+    private final EditableProperties editableProperty;
+    private static Map<String, String> checkerStrings;
```

Comment:
I don't think this field needs to be static. A slight rewrite of the initialization should allow you to make the field final instead.